lib/xdparser2.ts

```diff
@@ -290,19 +290,25 @@ export function xdParser(xd: string, strict = false, editorInfo = false): Crossw
   if (json.metapuzzle) json.metapuzzle.answer = json.metapuzzle.answer.trim()
 
   // Update the clues with position info and the right meta
+
   const positions = getCluePositionsForBoard(json.tiles)
   for (const keyClue of rawInput.clues) {
     const [_, clue] = keyClue
-    const arr = clue.dir === "A" ? json.clues.across : json.clues.down
+    const dirKey = clue.dir === "A" ? "across" : "down"
+    const arr = json.clues[dirKey]
+    const positionData = positions[clue.num]
+    const tiles = positionData.tiles[dirKey]!
 
-    const { answer, splits } = parseSplitsFromAnswer(clue.answer, json.meta.splitcharacter)
-    if (editorInfo && clue.metadata) clue.metadata["answer:unprocessed"] = clue.answer
+    const answerWithRebusSymbols = replaceWordWithSymbol(clue.answer, tiles, json.meta.splitcharacter)
+    const splits = parseSplitsFromAnswer(answerWithRebusSymbols, json.meta.splitcharacter)
```

Comment:
We need to parse splits from the symbol-substituted version of the answer otherwise the indexes will be off.